_posts/2021-12-20-authoring-a-url-to-start-a-workspace.adoc

```diff
@@ -0,0 +1,156 @@
+---
+title: Authoring a URL to start a workspace
+layout: post
+author: Florent Benoit
+description: >-
+  How to start a workspace by customizing the URL.
+categories: []
+keywords: ['GitHub', 'GitLab', 'Bitbucket', 'start', 'workspace']
+slug: /@florent.benoit/start-a-workspace-by-entering-a-url
+---
+
+// https://excalidraw.com/#json=Z4gT6H3GoZwJkfvVCajad,Qtltwy9uKpQOtosF8mh00g
+image::../../../assets/img/start-a-workspace/start-workspace-link.png[Start a workspace link,link="../../assets/img/start-a-workspace/start-workspace-link.png"]
+
+== Authoring a URL to start a workspace
+
+As a user, you can create a workspace containing a clone of a Git repository by entering:
+
+* a custom URL in the address bar of your browser.
+* the URL of the Git repository on the {prod} dashboard page.
+
```

Comment:
```suggestion

== Authoring a URL to start a workspace

As a user, you can create a workspace using a direct URL rather than using the user dashboard. 
This blog post describes how to author a URL to start a workspace.

To use a URL to start a workspace you can choose to:

* Paste the URL in the address bar of your browser.
* Distribute the URL as a link (in an onboarding procedure, in a pull request comment).

// https://excalidraw.com/#json=Z4gT6H3GoZwJkfvVCajad,Qtltwy9uKpQOtosF8mh00g
image::../../../assets/img/start-a-workspace/start-workspace-link.png[Authoring a URL to start a workspace,link="../../assets/img/start-a-workspace/start-workspace-link.png"]
```

Reply:
* Decouple authoring the URL / using the URL.
* Add the image after the lead in sentence (it has more context).